sunpy/net/scraper.py

```diff
@@ -274,9 +275,10 @@ def _httpfilelist(self, timerange):
                     opn.close()
             except HTTPError as http_err:
                 # Ignore missing directories (issue #2684).
-                if http_err.code == 404:
-                    continue
```

Comment:
Hmm, looking at this, I wonder if we keep the old behaviour for 404 at least.

Reply:
Sure but if we make it break where before it didn't break. That isn't an improvement.

Reply:
Yeah, I think we want to log the 404 as a debug but not have it fail.